### ISSUE TITLE
no documents get synced down

### ISSUE THREAD

**Issue by temharay** [pouchdb/pouchdb#8151]
### Issue
I was working on pouchdb 6 but due to changes in the envrionment, I had to upgrade to 7.2.1 (well now seeing it is 7.2.2 but i been at this bug for two days ) and all the way decided to use pouchdb-browser (the issue started with pouchdb latest version though). Now after syncing started, I can see connection was established with couch but no document gets synced down at all. In my network tab, I can see something like this:

`{"committed_update_seq":354578181,"compact_running":false,"db_name":"db","disk_format_version":0,"instance_start_time":1595855226634284,"purge_seq":0,"state":"Online","update_seq":354578181}`

and all the network requests are 200 or 204 but local db is empty. 

THe same code works well if I put a string url (instead of a pouchdb object) with basic auth, instead of cookies.

Looking at the replication events, only paused event is logged (with info being undefined). I am using the following code:

```
    this.local.replicate
    .from(this.remote, {
      live: true,
      retry: true
    }).on('change', function (change) {
      // yo, something changed!
      console.log(11330, ' changes ', change)
    }).on('paused', function (info) {
      // replication was paused, usually because of a lost connection
      console.log('paused ', info)
    }).on('active', function (info) {
      // replication was resumed
      console.log('active ', info)
    }).on('error', function (err) {
      // totally unhandled error (shouldn't happen)
      console.log(1339, err)
    });
```

My remote is defined as:

```
    this.remote = new PouchDB(
      'https://..../db',
      {
        fetch: (url, opts) => {
          fetch(url, { ...opts, credentials: "include" });
        }
      },
      {
        skip_setup: true
      }
    );
```
while the local pouch is:

```
    this.local = new PouchDB("data");
```

I have tried  cache : false when creating the local database but it didnt help

I can't go back to older version of pouch since deployment is not working for certain old libraries but i was hoping the change wouldn't affect syncing.

UPDATE: Restarted all and now I get his error on the pause listner:

```
CustomPouchError {stack: "TypeError: Cannot read property 'then' of undefine…ernal:///./node_modules/argsarray/index.js:16:18)", message: "Cannot read property 'then' of undefined", result: {…}}
message: "Cannot read property 'then' of undefined"
```

### Info
- Environment: (browser / angula 5)
- Platform: (Chrome)
- Adapter: (IndexedDB)
- Server: (CouchDB)

**Comment by sammysium** [comment]
@nolanlawson any idea please? Kinda frustrating.

**Comment by temharay** [comment]
solved it with 7.0..0 s fetch example:

```
     new PouchDB('url', {
  fetch: function (url, opts) {
    return PouchDB.fetch(url, opts, { credentials : "include"});
  }
});
```
